src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts

```diff
@@ -34,21 +34,18 @@ export class EditSpaceDescriptionWidgetComponent implements OnInit {
     private spaceNamespaceService: SpaceNamespaceService,
     private collaboratorService: CollaboratorService
   ) {
-    spaces.current
+    userService.loggedInUser.subscribe(val => this.loggedInUser = val);
```

Comment:
Sorry for going back and forth on this. Can we have either all setup in the constructor, or all setup in ngOnInit? Either way is fine.

Reply:
Ah yeah sounds good, I'll move the userService subscription to the init.